mozci/task.py

```diff
@@ -264,6 +264,51 @@ def errors(self):
             self._load_errorsummary()
         return self._errors
 
+    @property
+    def configuration(self):
+        SUITES = (
```

Comment:
Can you move this to a global? Possibly even in a utility file?

Reply:
Done with b2f644e.